quattor/types/hardware.pan

```diff
@@ -20,9 +20,12 @@ type structure_ram = {
 }
 type structure_cpu = {
     include structure_annotation
-    "speed"  : long # "CPU clock speed in MHz"
-    # Number of cores on each CPU chip, defaults to 1.
+    @{ desc = CPU clock speed in MHz }
+    "speed"  : long
+    @{ desc = Number of cores on each CPU chip }
     "cores" : long(1..) = 1
+    @{ desc = Number of execution threads on each CPU chip }
```

Comment:
@jrha I tend to agree with @stdweird. I understand that this can be a useful information but for me this is a site configuration choice, not something related to the HW characteristics. I think we should restrict this part of the schema to what is related to the HW description. What about a site-specific variable to track this kind of information? (this is in fact how it works in the templates of the template library that configures Torque/MAUI).